.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+      interval: "daily"
+    open-pull-requests-limit: 10
+    labels:
+      - "type/Dependencies"
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    labels:
+      - "type/Dependencies"
+      - "type/Github Actions"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)